plugins/filter_kubernetes/kube_conf.c

```diff
@@ -75,7 +75,19 @@ struct flb_kube *flb_kube_conf_create(struct flb_filter_instance *ins,
 
     /* Get Kubernetes API server */
     url = flb_filter_get_property("kube_url", ins);
-    if (!url) {
+
+    if (ctx->use_kubelet) {
+        ctx->api_host = flb_strdup(FLB_KUBELET_HOST);
```

Comment:
what does `flb_strdup` why is it needed here?

Reply:
flb_strdup() just duplicates an already existing SDS string.
This one is basically follow the style of existing code from this line: https://github.com/fluent/fluent-bit/blob/master/plugins/filter_kubernetes/kube_conf.c#L79